icons/qt/qt-original.svg

```diff
@@ -0,0 +1 @@
+<svg width="128" height="128" xmlns="http://www.w3.org/2000/svg"><g stroke="null"><path fill="#41CD52" d="M115.861 17H17.577L-.242 34.688v76.226H110.39l17.818-17.687V17H115.86zm0 0"/><path fill="#FFF" d="M47.567 90.531c-8.45 0-14.36-2.258-17.687-6.777-3.327-4.52-4.99-11.691-4.99-21.426 0-9.777 1.706-17.035 5.076-21.773 3.37-4.739 9.281-7.13 17.601-7.13 8.36 0 14.227 2.348 17.554 7.087 3.37 4.691 5.033 11.949 5.033 21.773 0 6.473-.699 11.688-2.054 15.645-1.403 3.996-3.635 7.039-6.788 9.125l6.831 10.91-8.364 3.867-7.222-11.777c-1.05.347-2.714.476-4.99.476zM37.015 77.973c1.837 3.129 5.385 4.738 10.596 4.738 5.207 0 8.711-1.566 10.549-4.652 1.797-3.086 2.714-8.344 2.714-15.688 0-7.387-.917-12.734-2.801-16.082-1.88-3.344-5.342-4.996-10.462-4.996-5.08 0-8.581 1.652-10.51 4.996-1.88 3.348-2.844 8.649-2.844 15.996 0 7.3.92 12.559 2.758 15.688zm0 0M101.504 55.766H90.296v17.773c0 3.305.26 5.477.742 6.563.482 1.086 1.75 1.609 3.722 1.609l6.653-.262.395 7.04c-3.634.695-6.392 1.042-8.316 1.042-4.599 0-7.752-1.043-9.458-3.129-1.707-2.086-2.584-6.039-2.584-11.863V55.81h-6.171v-7.52h6.214V36.598h8.803v11.691h11.208v7.477zm0 0"/></g></svg>
```

Comment:
This will fix the workflow issues mentioned in the comment that the bot made.
```suggestion
<svg width="128" height="128" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><g stroke="null"><path fill="#41CD52" d="M115.861 17H17.577L-.242 34.688v76.226H110.39l17.818-17.687V17H115.86zm0 0"/><path fill="#FFF" d="M47.567 90.531c-8.45 0-14.36-2.258-17.687-6.777-3.327-4.52-4.99-11.691-4.99-21.426 0-9.777 1.706-17.035 5.076-21.773 3.37-4.739 9.281-7.13 17.601-7.13 8.36 0 14.227 2.348 17.554 7.087 3.37 4.691 5.033 11.949 5.033 21.773 0 6.473-.699 11.688-2.054 15.645-1.403 3.996-3.635 7.039-6.788 9.125l6.831 10.91-8.364 3.867-7.222-11.777c-1.05.347-2.714.476-4.99.476zM37.015 77.973c1.837 3.129 5.385 4.738 10.596 4.738 5.207 0 8.711-1.566 10.549-4.652 1.797-3.086 2.714-8.344 2.714-15.688 0-7.387-.917-12.734-2.801-16.082-1.88-3.344-5.342-4.996-10.462-4.996-5.08 0-8.581 1.652-10.51 4.996-1.88 3.348-2.844 8.649-2.844 15.996 0 7.3.92 12.559 2.758 15.688zm0 0M101.504 55.766H90.296v17.773c0 3.305.26 5.477.742 6.563.482 1.086 1.75 1.609 3.722 1.609l6.653-.262.395 7.04c-3.634.695-6.392 1.042-8.316 1.042-4.599 0-7.752-1.043-9.458-3.129-1.707-2.086-2.584-6.039-2.584-11.863V55.81h-6.171v-7.52h6.214V36.598h8.803v11.691h11.208v7.477zm0 0"/></g></svg>

```

Reply:
Thanks! I applied this fix.